### PR TITLE
Update public_data.md

### PR DESCRIPTION
**Pull request by annaroyer**
@emhickmann

### CODE REVIEW

File: m4_sessions/1804inning/public_data.md [hunk 1]
```diff
@@ -5,12 +5,56 @@ Anna Royer
...
-Name of Session: 
+#### Name of Session: Using Public Data for Civic Tech
 
-Session Description (Why should students come to this session):
+#### Session Description (Why should students come to this session):
+Discussing and Engaging with material and others on what civic tech and public data are and how to use them to solve problems irl.
 
-Session Outcomes (What will students learn/leave with?):
+#### Session Outcomes (What will students learn/leave with?):
+* Students will be emboldened to become involved in Civic Tech projects
```
Comment (emhickmann) [inline m4_sessions/1804inning/public_data.md:14]: Nice vocab here!

File: m4_sessions/1804inning/public_data.md [hunk 2]
```diff
@@ -5,12 +5,56 @@ Anna Royer
...
+* Students will have concrete resources to becoming involved in Civic Tech
+* Students will brainstorm ideas of ways to become involved in Civic Tech
+ 
+#### Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+**Ice Breakers** (7 minutes)
+TBD
 
-Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+**Intro** (10 minutes)
+* What is Civic Tech?
```
Comment (emhickmann) [inline m4_sessions/1804inning/public_data.md:24]: Slides?  Key talking points?  Think about how to break this up and have each facilitator take part of it.

File: m4_sessions/1804inning/public_data.md [hunk 3]
```diff
@@ -5,12 +5,56 @@ Anna Royer
...
+* Real examples of how people have used and are using public data for civic tech
+* What classifies data as "public"
+* What is an API? 
+
+**Open Questions**(5 minutes)
+ 
+**Breakout** (7 minutes)
+(4 small groups: each group discusses one of the following questions)
```
Comment (emhickmann) [inline m4_sessions/1804inning/public_data.md:32]: This is great. Let me know if you want chart paper and markers for each group to chart their discussion

File: m4_sessions/1804inning/public_data.md [hunk 4]
```diff
@@ -5,12 +5,56 @@ Anna Royer
...
+
+**Breakout**(10 minutes)
+(4 small groups)
+ * Think of a problem that could be solved with public data
+ * How you might attack the problem?
+
+**Whole Group Share**(10 minutes)
+  
+**Breakout**(10 minutes)
+(4 small groups: each group is given a civic tech project to research)
```
Comment (emhickmann) [inline m4_sessions/1804inning/public_data.md:48]: Cool!

File: m4_sessions/1804inning/public_data.md [hunk 5]
```diff
@@ -5,12 +5,56 @@ Anna Royer
...
+ * Civic tech projects (TBD)
+ * What problem were they trying to solve?
+ * Who does it help and how?
+ * What data sources did they use?
+ * Is their solution effective and / or succesful?
+ * Who are the contributers -> how has the work been shared?
+ 
+**Whole Group Share**(10 minutes)
+
+**Wrap It Up**
```
Comment (emhickmann) [inline m4_sessions/1804inning/public_data.md:58]: Be more specific here...how will you wrap it up?  Making commitments?  Telling people to join Palhka? What action steps?

File: m4_sessions/1804inning/public_data.md [hunk 6]
```diff
@@ -5,12 +5,56 @@ Anna Royer
 Sabrina Kennedy
 Jared Eklin
 
-Name of Session: 
+#### Name of Session: Using Public Data for Civic Tech
 
-Session Description (Why should students come to this session):
+#### Session Description (Why should students come to this session):
+Discussing and Engaging with material and others on what civic tech and public data are and how to use them to solve problems irl.
```
Comment (emhickmann) [inline m4_sessions/1804inning/public_data.md:11]: Additional resource: https://github.com/turingschool/gear-up/blob/master/m4_sessions/1803-inning/Public_Data_To_Change_The_World.md

**Review by emhickmann — COMMENTED** [review]
@annaroyer Great start!!  This is awesome.  Just work through timing and decide who is going to lead which part.  Get that content in a good spot and you will have a great session.

File: m4_sessions/1804inning/public_data.md (hunk 3 above)
Comment (annaroyer) [reply m4_sessions/1804inning/public_data.md:32]: Chart paper and markers would be great for this discussion. Thank you!

**Pull request merged**